Doc upgrade guide (#6231)
cyberbotics/webots: docs/guide/from-2021b-to-2022a.md

# Upgrade from R2021b to R2022a

Starting from the R2022a version, all geometries, devices, and PROTOs that come with Webots use the _FLU_ axis orientation (x-**F**orward, y-**L**eft, and z-**U**p).
We introduced the update to be consistent with ROS (see [REP 103](https://www.ros.org/reps/rep-0103.html)) and other robotics systems.
If your Webots world uses Webots geometries, devices, and PROTOs then you will probably need to adapt your world.

| ![spot2021b.png](images/spot_nue.png) | ![spot2022a.png](images/spot_enu.png) |
|:--:|:--:|
| *spot.wbt in 2021b: spot is RUB and the world is NUE.* | *spot.wbt in 2022a: spot is FLU and the world is ENU.* |

> For a comprehensive list of the changes, please refer to the [Change Log](../reference/changelog-r2022.md).

Reviewer: ```suggestion
Changes breaking the backward compatibility are listed in bold in the [Change Log](../reference/changelog-r2022.md).
```

| [Viewpoint](../reference/viewpoint.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, \frac{\pi}{2})"> | A1 |
| [Track](../reference/track.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, \frac{\pi}{2})"> | A2 |
| [Pen](../reference/camera.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, 0)"> | A2 |
| [Emitter](../reference/emitter.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, -\frac{\pi}{2})"> | A2 |
| [Receiver](../reference/receiver.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, -\frac{\pi}{2})"> | A2 |
| [Connector](../reference/connector.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, -\frac{\pi}{2})"> | A2 |
| [TouchSensor](../reference/touchsensor.md) | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, -\frac{\pi}{2})"> | A2 |

> | Webots PROTOs | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, \frac{\pi}{2})">, but there are numerous exceptions to this rule as the protos did not followed any specific convention any now | C |

Reviewer 2: ```suggestion
| Webots PROTOs | <img src="https://render.githubusercontent.com/render/math?math=\color{gray}\large(-\frac{\pi}{2}, 0, \frac{\pi}{2})">, but there are numerous exceptions to this rule as the PROTOs did not followed any specific convention any now | C |
```
